bitcoin/src/blockdata/transaction.rs

```diff
@@ -183,15 +183,15 @@ impl core::str::FromStr for OutPoint {
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 #[cfg_attr(feature = "serde", serde(crate = "actual_serde"))]
 pub struct TxIn {
-    /// The reference to the previous output that is being used an an input.
+    /// The reference to the previous output that is being used as an input.
     pub previous_output: OutPoint,
     /// The script which pushes values on the stack which will cause
     /// the referenced output's script to be accepted.
     pub script_sig: ScriptBuf,
     /// The sequence number, which suggests to miners which of two
     /// conflicting transactions should be preferred, or 0xFFFFFFFF
     /// to ignore this feature. This is generally never used since
-    /// the miner behaviour cannot be enforced.
+    /// the miner behavior cannot be enforced.
```

Comment:
Not gonna hold up this PR over it, but I guess we should make a crate-wide decision as to what English dialect we're using. Currently there are several instances of both `behavior` and `behaviour` in the crate.

Reply:
I try to favour US English in `rust-bitcoin` but sometimes I forget (I use British English in my own writing usually as is typical of Australians). I think we should pick a single style. FWIW I have my editor set to US English for this reason.

Reply:
ACK for US English from me too. (Personally I tend to use Canadian English which is a poorly-defined hybrid of US and British.)

Reply:
Hmm interesting, I didn't realize behaviour was the british spelling of the word.  I usually default to `set spell spelllang=en_us` in vim which explains why it was flagged.  I'd imagine The Rust language also defaults to en_us.

> ACK for US English from me too. (Personally I tend to use Canadian English which is a poorly-defined hybrid of US and British.)

+1 for us_en as well.

> Not gonna..

Is "gonna" an example of poorley defined Canadian English or US English :)

Reply:
> Is "gonna" an example of poorley defined Canadian English or US English :)

Lol! I actually have no idea where "gonna" is and isn't used. I haven't been everywhere in Canada and the US but nobody in the places I have been have raised their eyebrows at it.

In any case, we shouldn't use it in documentation since it's too slangy :)